packages.json

```diff
@@ -16,6 +16,7 @@
     "maintainer": "Aaron Wolen <aaron@tiledb.com>",
     "url": "https://github.com/single-cell-data/TileDB-SOMA",
     "available": true,
+    "branch": "*release",
```

Comment:
:+1: but as I am not a `remotes` user: we are certain this is right syntax?  I.e. it is not a wildcard match to, say `release-.*`  ?